modules/UI/videolayout/SmallVideo.js

```diff
@@ -609,19 +607,22 @@ SmallVideo.prototype.updateView = function() {
  * Updates the react component displaying the avatar with the passed in avatar
  * url.
  *
- * @param {string} avatarUrl - The uri to the avatar image.
  * @returns {void}
  */
-SmallVideo.prototype.avatarChanged = function(avatarUrl) {
+SmallVideo.prototype.avatarChanged = function() {
```

Comment:
Should this be changed to something like initializeAvatar?

Reply:
Yep, I renamed it, thanks!